account_asset_management/models/account_asset.py

```diff
@@ -162,6 +162,11 @@ class AccountAsset(models.Model):
              # "  * Ending Date: Choose the time between 2 depreciations "
              # "and the date the depreciations won't go beyond."
     )
+    days_calc = fields.Boolean(
+        string='Calculate by days',
+        default=False,
```

Comment:
@Saran440 No need to define `False`, Boolean always Default False 
https://www.odoo.com/documentation/10.0/reference/orm.html#odoo.fields.Field